cylc/flow/unicode_rules.py

```diff
@@ -197,8 +237,10 @@ def not_equals(string):
 
     """
     return (
-        re.compile(rf'^(?!{re.escape(string)}$).*$'),
-        f'cannot be: ``{string}``'
+        re.compile(rf'^(?!^{_re_format_list(strings)}$).*$'),
+        # rf'^(?!{_re_format_list(strings)})$'),
+        formatted  _human_format_list([f'``{s}``' for s in strings])
+        f'cannot be: {formatted}'
```

Comment:
D'oh I made a mess of this the first time round
```suggestion
        'cannot be: ' + _human_format_list([f'``{s}``' for s in strings])
```